### ISSUE TITLE
Recent: TV Not Turning On, Just "On" Affected

### ISSUE THREAD

**Issue by shanemcw** [merdok/homebridge-webos-tv#544]
Recently, about a week ago, a controlled TV no longer turns on via the Home app. The TV can be turned off, the inputs changed, etc. via the Home app if it is on, but not "on" if it is off. On the Home app, via Siri on a manual set of the TV to be "on," Home reports that it is on, and automation triggered by the "on" executes, but the TV is not physically on.

Recording this data point in case others find this condition.

**Comment by shanemcw** [comment]
It looks like a newly introduced bug in WebOS. In my context, the "turn on with WiFi" setting is "on" in settings (it looks to be specific to just the "on" function), but I can confirm that the official remote app that simulates the remote can also turn off the TV but not back on. So, I confirmed this is not a plugin issue but perhaps a bug preventing general power on control via Home regardless of that setting activated.

**Comment by merdok** [comment]
I am pretty sure that it has something  to do with your network. The turn on command is sent to the TV with the standard Wake on LAN / Wake on WLAN command. That means the TV needs to be in the same network as your homebridge instance and the power saving features on the tv need to be off.

**Comment by shanemcw** [comment]
I know, _weird_ right? Confirmed power savings are still off, confirmed is still on same network, been working for over a year, just stopped powering on a week or so ago. Started with scenes—a scene would do everything—including trying to turn the TV on—and automations get the "TV turned on" event, but TV not turned on. Weird the LG Remote iPhone app cannot turn it on either, just turn it off and everything else. Confirmed the "allow other apps to turn on the TV" is still on.

**Comment by merdok** [comment]
Yeah well you have to investigate your local setup and check what could be the cause that the wake on lan packets do not arrive at your tv.

**Comment by wondertwins** [comment]
I am having this issue as well.

**Comment by merdok** [comment]
@wondertwins the issue will probably be more or less the same in your case. The WOL packets might not arrive at the TV...

**Comment by wondertwins** [comment]
@merdok yep, it was due to my vpn configuration. Even though I have homebridge setup to use en0 Network Interface, the WOL packets was not getting through.

**Comment by wondertwins** [comment]
@shanemcw Try setting your broadcast address in advanced settings. I have mine now set to 192.168.1.255 and its working now.

**Comment by mrfatboy** [comment]
> @shanemcw Try setting your broadcast address in advanced settings. I have mine now set to xxx.xxx.xxx.255 and it's working now.

I recently had the same issue with the plugin. Did the same fix as you and it is now working again.

**Comment by shanemcw** [comment]
Thanks!!!
